tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -300,7 +330,7 @@ def test_logs_surfacing_scheduler_basic_flow(self, cluster):
         cluster_logs = "\n".join(cluster_logs)  # make logs list into one string
 
         assert (
-            "Performing cluster checks: potentially sending to Den, surfacing logs to Den or updating autostop."
+            f'HTTP Request: POST {api_server_url}/resource/{cluster_uri}/cluster/status "HTTP/1.1 200 OK"'
```

Comment:
how reliable is this response (ex: if the logger format changes, can we rely on this?) maybe we can confirm this request works more generically

Reply:
@jlewitt1 yeah, good point. I updated the test, to make sure that the status was updated in the last minute (DEFAULT_STATUS_CHECK_INTERVAL). If the cluster periodic loop running as it should, the status should be updated every DEFAULT_STATUS_CHECK_INTERVAL minute(s).